app/assets/javascripts/widgets/wysiwyg.js

```diff
@@ -0,0 +1,14 @@
+$(document).ready(function(){
+  var anchorLinks = $('h3.anchorLinks a, a[href="#top"]');
+  var navHeight = $("#cu_nav").outerHeight();
```

Comment:
There is a convention (which I like and use myself) of starting any var that is a jQuery element object or collection with `$`. So that this would be `$navHeight`. Should we adopt that as part of our style guide?